code/modules/cargo/packs/organic.dm

```diff
@@ -3,18 +3,35 @@
 // If you add something to this list, please group it by type and sort it alphabetically instead of just jamming it in like an animal
 // cost = 700- Minimum cost, or infinite points are possible.
 //////////////////////////////////////////////////////////////////////////////
-//////////////////////////// Organic /////////////////////////////////////////
+//////////////////////////////// Organic /////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////////
 
 /datum/supply_pack/organic
 	group = "Food & Hydroponics"
 	crate_type = /obj/structure/closet/crate/freezer
 
 //////////////////////////////////////////////////////////////////////////////
-/////////////////////////////// Food /////////////////////////////////////////
+//////////////////////////////// Meals ///////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////////
 
-/datum/supply_pack/organic/candy/randomised
+/datum/supply_pack/organic/combomeal2
+	name = "Burger Combo #2"
+	desc = "We value our customers at the Greasy Griddle, so much so that we're willing to deliver -just for you.- This combo meal contains two burgers, a soda, fries, a toy, and some chicken nuggets."
+	cost = 3200
+	contains = list(/obj/item/reagent_containers/food/snacks/burger/bigbite,
+					/obj/item/reagent_containers/food/snacks/burger/cheese,
+					/obj/item/reagent_containers/food/snacks/fries,
+					/obj/item/reagent_containers/food/condiment/pack/ketchup,
+					/obj/item/reagent_containers/food/condiment/pack/ketchup,
+					/obj/item/reagent_containers/food/snacks/nugget,
+					/obj/item/reagent_containers/food/snacks/nugget,
+					/obj/item/reagent_containers/food/snacks/nugget,
+					/obj/item/reagent_containers/food/snacks/nugget,
+					/obj/item/toy/plush/random)
+	crate_name = "combo meal w/toy"
+	crate_type = /obj/structure/closet/crate/wooden
+
+/datum/supply_pack/organic/randomizedcandy
```

Comment:
Please make the candies a subtype of `/randomized`. And move the num_contained variable to the `/randomized` type.